spec/fixtures/addons/text-inputs.scss

```diff
@@ -1,17 +1,17 @@
 @import "setup";
 
-#{$all-text-inputs} {
-  color: #ff0000;
+.all-text-inputs {
+  value: $all-text-inputs;
```

Comment:
:+1: I think there's perfectly valid times like this when bending the styleguide makes for better tests